text/0000-remove-namespacing.md

```diff
@@ -0,0 +1,50 @@
+- Start Date: 2015-12-03
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes we remove all namespacing in the Refract base specification
+while proposing we use `profile` links in its place.
+
+Note: this is not to be confused with the actual namespace documents we have
```

Comment:
I think if we move to profiles, it seems like they should be called such, though I'm definitely open to suggestions here.